pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1091,6 +1097,18 @@ def _get_strongest_unique_aps(self, access_points):
 
         return strongest_aps.values()
 
+    def _get_strongest_ap_for_ssid(self, access_points, ssid):
```

Comment:
I wonder, shouldn't this be provided by the module?

Reply:
To me it seems as too ad-hoc API at this point.